RecoTracker/MkFit/python/dumpMkFitGeometry.py

```diff
@@ -0,0 +1,48 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this usually goes to /test, not to /python

Reply:
moved